cabal-install/Distribution/Client/Types.hs

```diff
@@ -591,3 +591,19 @@ instance Monoid AllowNewer where
 instance Monoid AllowOlder where
   mempty  = AllowOlder mempty
   mappend = (<>)
+
+-- ------------------------------------------------------------
+-- * --write-ghc-environment-file
+-- ------------------------------------------------------------
+
+-- | Whether 'v2-build' should write a .ghc.environment file after
+-- success. Possible values: 'always', 'never', 'ghc8.4.4+' (the
+-- default; GHC 8.4.4 is the earliest version that supports
+-- '-pkg-env -').
```

Comment:
`-package-env`?

GHC-8.0.1 seems to support `-package-env`.

Am I missing something?

Reply:
FWIW, `-package-env` is also the only option with `env` in it which `ghc --show-options` knows.

Reply:
Fixed the comment. It's about `-package-env -` (note the dash), which disables package environments, was added in GHC 8.6, and then backported to 8.4.4. See https://ghc.haskell.org/trac/ghc/ticket/13753 and https://downloads.haskell.org/~ghc/8.4.4/docs/html/users_guide/8.4.4-notes.html#compiler.

Reply:
@23Skidoo thanks for clarification!